scripts/README.md

```diff
@@ -0,0 +1,39 @@
+# FLOSS Scripts
+FLOSS supports converting its output into scripts for various tools. Please see the render scripts in this directory.
+  
+Additionally, there is another [plugin for IDA](idaplugin.py) to allow FLOSS to automatically
+extract obfuscated strings and apply them to the currently loaded module in IDA.
+
+# Installation
+These scripts can be downloaded from the FLOSS [GitHub](https://github.com/mandiant/flare-floss) repository
+alongside the source, which is required for the scripts to run.
+To install FLOSS as source, see the documentation [here](../doc/installation.md).
+
+
+# Usage
+## Convert FLOSS output for use by other tools
+
+- Run FLOSS on the desired executable with the `-j` or `--json` argument to emit a JSON result
+and redirect it to a JSON file.
+
+    $ floss -j suspicious.exe > floss_results.json
+
+For Binary Ninja, IDA Pro, Ghidra or Radare2:
+- Run the script for your tool of choice by passing the result json file as an argument and
+redirect the output to a Python (.py) file.  
+
+Ghidra Example:
+
+    $ python render-ghidra-import-script.py floss_results.json > apply_floss.py
```

Comment:
```suggestion
    `$ python render-ghidra-import-script.py floss_results.json > apply_floss.py`
```

Reply:
But wouldn't this make it a one line code? I wanted it to look similar to the existing docs.
![image_2023-04-04_132859320](https://user-images.githubusercontent.com/126584775/229726756-e5a52914-23ee-4b1d-90be-e8e4b61ccf2e.png)